biogeophys/FatesHydroWTFMod.F90

```diff
@@ -713,9 +781,11 @@ subroutine set_wrf_param_cch(this,params_in)
     this%psi_max     = this%psi_from_th(this%th_max-tiny(this%th_max))
     this%dpsidth_max = this%dpsidth_from_th(this%th_max-tiny(this%th_max))
 
-    this%th_min      = min_theta_cch
-    this%psi_min     = this%psi_from_th(min_theta_cch+tiny(this%th_max))
-    this%dpsidth_min = this%dpsidth_from_th(min_theta_cch+tiny(this%th_max))
+    !this%th_min      = min_theta_cch
+    !this%psi_min     = this%psi_from_th(min_theta_cch+tiny(this%th_max))
+    this%psi_min     = min_psi_cch
+    this%th_min      = this%th_from_psi(min_psi_cch+tiny(this%th_max))
```

Comment:
what's the purpose of the tiny(this$th_max)? Wondering if this is an error, since the input to this function should be only water potential.

Reply:
originally, this tiny() function was used to prevent a problem with order of operations. Inside these functions we have "if statements" that compare theta and psi to these very psi_min and th_min thresholds we are calculating. By adding or subtracting a tiny, we were preventing the intering of logical blocks that were not yet initialized.  But with these changes, we actually don't really use these tiny's as much